src/deepsparse/loggers/function_logger.py

```diff
@@ -89,3 +90,23 @@ def log(self, identifier: str, value: Any, category: MetricCategories):
                 )
                 self._function_call_counter = 0
             self._function_call_counter += 1
+
+    def __str__(self):
+        def _indent(value):
+            return textwrap.indent(str(value), prefix="  ")
+
+        def _shorten(text, max_num_lines=10, placeholder="[...]"):
```

Comment:
All the proposed solutions involving text wrap (https://stackoverflow.com/questions/1166317/python-textwrap-library-how-to-preserve-line-breaks) sounded more complicated and less readable then this small function.